src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -462,11 +464,41 @@ trait FetchUriFormats {
   }
 }
 
+trait MesosFormats {
```

Comment:
This is not part of API, is it? As such it has nothing to do in the `api.v2` package.